app/models/role.rb

```diff
@@ -0,0 +1,39 @@
+# Model representing a user's role in a given course.
+class Role < ApplicationRecord
+  belongs_to :user
```

Comment:
Make this `belongs_to :human` (or the name you decide for the user subclass... human still feels not quite right but I don't really have a better idea... client? end_user? member? scholar? ... maybe discuss with David on Monday)